File: relationship-manager/customer-profile-service/src/main/java/com/bank/rm/profile/ProfileServiceApplication.java
package com.bank.rm.profile;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.bank.rm.profile", "com.bank.rm.common"})
public class ProfileServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(ProfileServiceApplication.class, args);
    }
}

File: relationship-manager/customer-profile-service/src/main/kotlin/com/bank/rm/profile/ProfileServiceApplication.kt
package com.bank.rm.profile

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.profile", "com.bank.rm.common"])
class ProfileServiceApplication

fun main(args: Array<String>) {
    runApplication<ProfileServiceApplication>(*args)
}
